package.json

```diff
@@ -20,6 +20,8 @@
   },
   "homepage": "https://github.com/acmucsd/membership-portal#readme",
   "dependencies": {
+    "@sendgrid/mail": "^6.4.0",
+    "async": "^3.1.0",
```

Comment:
don't need this dependency so remove it and make sure to update the lockfile as well